### PR TITLE
CN-3186: refactor reducers

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/redux/userAuth/userAuthReducer.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import ACTIONS from "../../constants/ACTIONS";
+
+/**
+ * TODO make it a function add JSDoc & desc for return
+ */
+const initialState = {
+  tokens: undefined, // TODO move to authReducer
```
Comment (fjovakaric) [inline app/redux/userAuth/userAuthReducer.js]: remove TODOs

File: app/redux/userAuth/userAuthReducer.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        },
+      };
+
+    default:
+      return state;
+  }
+};
+
+// TODO: should be moved to user model or util, thisi s a quickfix for CN-4320
+function mapProfile(userProfile) {
```
Comment (fjovakaric) [inline app/redux/userAuth/userAuthReducer.js]: move to user util

File: app/redux/userSecurity/userSecurityReducer.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  showVerifyScreen: false,
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+
+    case ACTIONS.SET_PIN_SUCCESS:
+      return {
+        ...state,
+        profile: {
```
Comment (fjovakaric) [inline app/redux/userSecurity/userSecurityReducer.js]: this is related to the user, not security

File: app/utils/api-util.js [hunk 4]
```diff
@@ -77,7 +77,7 @@ function initInterceptors() {
         };
       }
 
-      if (!req.url.includes("branch.io")) {
+      if (!req.url.includes("campaign.io")) {
```
Comment (fjovakaric) [inline app/utils/api-util.js]: should stay branch

File: app/utils/api-util.js [hunk 5]
```diff
@@ -98,7 +98,7 @@ function initInterceptors() {
           ...newRequest.headers,
           "Content-Type": "multipart/form-data",
         };
-      } else if (req.method === "post" && !req.url.includes("branch.io")) {
+      } else if (req.method === "post" && !req.url.includes("campaign.io")) {
```
Comment (fjovakaric) [inline app/utils/api-util.js]: should stay branch

File: app/utils/branch-util.js [hunk 6]
```diff
@@ -35,7 +35,7 @@ function initBranch() {
 }
 
 /**
- * Gets the id of the referral the user used for registrations. handles branch link and manual entering of code
+ * Gets the id of the referral the user used for registrations. handles campaign link and manual entering of code
```
Comment (fjovakaric) [inline app/utils/branch-util.js]: should stay branch

File: set-env.js [hunk 7]
```diff
@@ -16,7 +16,7 @@ const ALL_CONFIGS = {
 const ENV_FILES = {
   APP_JSON: 'app.json',
   CONSTANTS: 'constants.js',
-  BRANCH: 'branch.json',
+  BRANCH: 'campaign.json',
```
Comment (fjovakaric) [inline set-env.js]: should stay branch
